language/move-vm/types/src/loaded_data/runtime_types.rs

```diff
@@ -50,6 +50,7 @@ pub enum Type {
     U16,
     U32,
     U256,
+    Function,
```

Comment:
Hmm... no type parameters?

Reply:
So I intentionally hided them because the runtime already carried the type information so there's no need to duplicate it.